blazor/datagrid/batch-editing.md

```diff
@@ -338,4 +338,70 @@ Also, the add operation is handled while performing batch editing using the bool
     }
 }
 
+```
+
+## How to select a text in a cell when batch editing
+
+By default, when editing text in a particular cell in batch editing, the cursor will be placed at the end of the text. Instead of this behavior, the text can be changed in the selected state when editing. This can be achieved by the [OnCellEdit](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_OnCellEdit) and [RowSelected](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_RowSelected) events of the Grid.
```

Comment:
By default, when editing the text in a particular cell in batch editing, the cursor will be placed at the end of the text. Instead of this behavior, the text can be changed in the selected state when editing. This can be achieved by the [OnCellEdit](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_OnCellEdit) and [RowSelected](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_RowSelected) events of the Grid.